docs/section5/Rule5-47.md

```diff
@@ -0,0 +1,53 @@
+**Rule Assertion:** Sum of B-RMR infiltration.infiltration.air_leakage_rate = expected value.  
+**Appendix G Section:** Section G3.1-5(h) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** 
+
+  1. get_surface_conditioning_category()
+  2. get_zone_conditioning_category()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dict_b = get_surface_conditioning_category(B_RMR)`
+
+- Get zone conditioning category dictionary for B_RMR: `zone_conditioning_category_dict_b = get_zone_conditioning_category(B_RMR)`
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`
+
+      - For each surface in zone: `for surface_b in zone_b.surfaces:`
+
+        - Check if surface is regulated, add zone total area of building envelope to building total: `if scc_dict_b[surface_b.id] != "UNREGULATED": building_total_envelope_area += sum(surface.area for surface in zone_b.surfaces)`
+
+      - Check if zone is conditioned or semi-heated, add zone air leakage rate to building total: `if zone_conditioning_category_dict_b[zone.id] in [CONDITIONED RESIDENTIAL, CONDITIONED NON-RESIDENTIAL, CONDITIONED MIXED, SEMI-HEATED]: building_total_air_leakage_rate += zone_b.infiltration.air_leakage_rate`
+
+- Calculate the required baseline air leakage rate at 75Pa: `target_air_leakage_rate_b = 1.0 * building_total_envelope_area * 0.00047194745`
+
+**Rule Assertion:**  
+
+- Case 1: For B_RMR, if infiltration pressure difference is 75Pa and the total air leakage rate for conditioned and semi-heated zones is equal to the required baseline air leakage rate at 75Pa: `if ( B_RMR.ASHRAE229.infiltration_pressure_difference == 75 ) AND ( building_total_air_leakage_rate == target_air_leakage_rate_b ): PASS`
+
+- Case 2: Else if infiltration pressure difference is 75Pa for B_RMR and the total air leakage rate for conditioned and semi-heated zones is not equal to the required baseline air leakage rate at 75Pa: `if ( B_RMR.ASHRAE229.infiltration_pressure_difference == 75 ) AND ( building_total_air_leakage_rate != target_air_leakage_rate_b ): FAIL`
+
+- Case 3: Else if infiltration pressure difference is 0Pa for B_RMR and the total air leakage rate for conditioned and semi-heated zones is equal to the required baseline air leakage rate at 0Pa: `if ( B_RMR.ASHRAE229.infiltration_pressure_difference == 0 ) AND ( building_total_air_leakage_rate == target_air_leakage_rate_b * 0.112 ): PASS`
```

Comment:
I am not familiar with this rule for air leakage rate @ 0 Pa. What App G requirement does this refer to?

Reply:
@supriyagoel this is air leakage at wind pressure (no pressurization). Let's discuss at the next RDS call.

Reply:
have two options- 1. a flag in the RMR which states that infiltration rate specified as wind pressure or 2. add a note in the RMR which states that a value of '0' indicates infiltration specified at wind pressure.

Reply:
It seems that if we want to verify both (A) leakage at 75Pa, as prescribed in Table G3.1 #5 item (b) in the proposed column and item (h) in the baseline column, and (B) leakage at wind pressure as prescribed in G3.1.1.4, we'd need to rename the current infiltration.air_leakage_rate to something like infiltration.tested_air_leakage_rate, and add infiltration.air_leakage_at_wind_pressure. 

For the baseline, RCT will only verify infiltration.air_leakage_at_wind_pressure. For proposed, we would also verify infiltration.air_leakage_at_wind_pressure, and if the entered value is not equivalent to the default of 0.6 CFM/SF @75Pa converted to the rate at wind pressure per G3.1.1.4, will additionally check input for infiltration.tested_air_leakage_rate.